lib/workers/pr/body/index.ts

```diff
@@ -13,7 +13,9 @@ import { getPrUpdatesTable } from './updates-table';
 function massageUpdateMetadata(config: BranchConfig): void {
   config.upgrades.forEach((upgrade) => {
     /* eslint-disable no-param-reassign */
-    const { homepage, sourceUrl, sourceDirectory, changelogUrl } = upgrade;
+    const { sourceUrl, sourceDirectory } = upgrade;
+    const homepage = upgrade.homepage as string;
+    const changelogUrl = upgrade.changelogUrl as string;
```

Comment:
Maybe we should proper type them on `BranchUpgrade`  🤔

Reply:
Just added the fields to `BranchUpgradeConfig`